### PR TITLE
Adding Contrast Curve Code and its tests

### PR DESCRIPTION
**Pull request by holdengill**
I added my contrast curve code as Contrast.py in the src/simmer directory, and added its tests under src/simmer/tests titled 'test_contrast.py'. There was a lot of messing around with git so the log of commits is very messy but there should only be the two added files.

### CODE REVIEW

File: src/simmer/tests/test_contrast.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from astropy.io import fits
+import math
+from numpy import random
+import photutils
+from photutils import datasets
+from astropy.table import Table
+from Contrast import ConCur as cc
+from Contrast import twoD_weighted_std as wstd
+from Contrast import find_best_center
+import pdb
```
Comment (arjunsavel) [inline src/simmer/tests/test_contrast.py:19]: the pdb line can be removed — we won't need it for testing purposes.

File: src/simmer/tests/test_contrast.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import yaml
+import unittest
+import os
+import sys
+sys.path.append('Users/holdengill/Desktop')
```
Comment (arjunsavel) [inline src/simmer/tests/test_contrast.py:5]: we should likely remove this line!

**Review by arjunsavel — CHANGES_REQUESTED** [review]
This looks fantastic! A few things to address before I merge:

- [ ] I made a few comments to delete a few import / sys lines. Once those are addressed, things should be ready to go.
- [ ] Would you mind adding photutils to the setup.py file in this PR? Otherwise, that package isn't installed on the virtual machine that runs the tests.
- [ ] Before you make those changes and push, please install[ pre-commit](https://pre-commit.com/). Anytime you make a commit to this repo, it'll automatically apply the package's code style / formatting configurations (after which you'll have to git add and commit any files it modified again).

Please let me know if you have any questions — and thanks so much for all your hard work!

**Review by holdengill — COMMENTED** [review]
removed some unnecessary imports, added photutils to setup.py, downloaded pre-commit.

**Pull request merged**